src/schemas/components/SchemaFormPropertyArray.vue

```diff
@@ -58,22 +70,29 @@
 
     return property.default ?? null
   }
+
+  function isFirstIndex(index: number): boolean {
+    return index === 0
+  }
+
+  function isLastIndex(index: number): boolean {
+    return index === (props.value?.length ?? 0) - 1
+  }
 </script>
 
 <style>
 .schema-form-property-array .p-draggable-list__item { @apply
   block
 }
 
-.schema-form-property-array__item { @apply
-  grid
-  gap-2
-  items-start;
-  grid-template-columns: min-content 1fr min-content;
+.schema-form-property-array .p-draggable-list > .p-button {
+  margin-left: 30px;
 }
 
-.schema-form-property-array__handle { @apply
-  cursor-grab
-  mt-2
+.schema-form-property-array__empty { @apply
+  text-subdued
+  text-sm
+  italic;
+  margin-left: 30px;
```

Comment:
I'm slightly surprised this worked out to 30px and didn't land on the 8pt grid, but it's an eyeball number regardless.

Reply:
This was, special lol. I wanna revisit this at the base component to clean this up. This this keeps me moving for now and it looks fine. Its the width of the drag icon itself plus 8px for gap I believe.